README.md

```diff
@@ -61,5 +67,5 @@ If you want to contribute, see [CONTRIBUTING.md](CONTRIBUTING.md).
 
 This work is licensed under a [Creative Commons Attribution 4.0 International License](
 https://creativecommons.org/licenses/by/4.0/). The implementation uses the concepts of the
```

Comment:
instead of "uses the concepts" perhaps : "is based on the specifications of"

Reply:
done